src/pages/workspaces/workspace/Analyses.js

```diff
@@ -259,7 +261,7 @@ const Analyses = _.flow(
   )(async () => {
     const rawAnalyses = await Ajax(signal).Buckets.listAnalyses(googleProject, bucketName)
     const notebooks = _.filter(({ name }) => _.endsWith(`.${tools.Jupyter.ext}`, name), rawAnalyses)
-    const rmds = _.filter(({ name }) => _.endsWith(`.${tools.RStudio.ext}`, name), rawAnalyses)
+    const rmds = _.filter(({ name }) => _.includes(getExtension(name), tools.RStudio.ext), rawAnalyses)
```

Comment:
does this return both .Rmd and .R files? If so may be worth updating the variable name to something like `rAnalyses`